cmd/cofidectl/cmd/trustzone/trustzone.go

```diff
@@ -156,6 +156,18 @@ func (c *TrustZoneCommand) GetAddCommand() *cobra.Command {
 
 			bundleEndpointProfile := trust_zone_proto.BundleEndpointProfile_BUNDLE_ENDPOINT_PROFILE_HTTPS_SPIFFE
 
+			newTrustZone := &trust_zone_proto.TrustZone{
+				Name:                  opts.name,
+				TrustDomain:           opts.trustDomain,
+				JwtIssuer:             &opts.jwtIssuer,
+				BundleEndpointProfile: &bundleEndpointProfile,
+			}
+
+			_, err = ds.AddTrustZone(newTrustZone)
+			if err != nil {
+				return fmt.Errorf("failed to create trust zone %s: %w", newTrustZone.Name, err)
```

Comment:
Nit: could this be `newTrustZone.GetName()` as we do with `newCluster.GetName()`?